frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue

```diff
@@ -62,9 +62,12 @@
             </VCol>
             <VCol :cols="8">
               <VSelect
+                v-model="playbackManager.playbackSpeed"
                 density="comfortable"
-                disabled
-                hide-details />
+                hide-details
+                :items="playbackSpeed"
+                item-title="title"
+                item-value="speed" />
```

Comment:
What do you think about using a [Combobox](https://vuetifyjs.com/en/components/combobox/#multiple-combobox) instead? So we provide a good UX by giving a predefined selection of speeds but also freedom of choice in a "single package".

Instead of 1x, I'd add a "Normal" label to that option (just like YouTube does).

In that case, since you're going to use vue-i18n's t function for translating, you must wrap this array in a Vue's ``computed`` so language changes are reactive.

Reply:
I think we should keep using VSelect instead of the ComboBox, because after trying out ComboBox, I've encountered some validation(?) issues and changing the speed is no longer possible for some reason.
If we just keep the predefined speeds, then we don't have to deal with validating the user input.
If you want to try it yourself, here's a patch I made:
```
diff --git a/frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue b/frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue
index bcaada01..3ae26b26 100644
--- a/frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue
+++ b/frontend/src/components/Buttons/Playback/PlaybackSettingsButton.vue
@@ -61,13 +61,14 @@
               <label>{{ $t('speed') }}</label>
             </VCol>
             <VCol :cols="8">
-              <VSelect
-                v-model="playbackManager.playbackSpeed"
+              <VCombobox
+                v-model.number="playbackManager.playbackSpeed"
                 density="comfortable"
                 hide-details
                 :items="playbackSpeed"
                 item-title="title"
-                item-value="speed" />
+                item-value="speed"
+                :rules="[validationRule]" />
             </VCol>
           </VRow>
           <VRow align="center">
@@ -90,34 +91,44 @@
 </template>

 <script setup lang="ts">
+import { computed } from 'vue';
 import { playbackManager } from '@/store/playback-manager';
 import { playerElement } from '@/store/player-element';
+import { isNumber } from '@/utils/validation';

 const menuModel = defineModel<boolean>();
-const playbackSpeed = [
-  {
-    title: '0.5x',
-    speed: 0.5
-  },
-  {
-    title: '0.75x',
-    speed: 0.75
-  },
-  {
-    title: '1x',
-    speed: 1
-  },
-  {
-    title: '1.25x',
-    speed: 1.25
-  },
-  {
-    title: '1.5x',
-    speed: 1.5
-  },
-  {
-    title: '2x',
-    speed: 2
-  }
+const playbackSpeed = computed(() => {
+  return [
+    {
+      title: '0.5x',
+      speed: 0.5
+    },
+    {
+      title: '0.75x',
+      speed: 0.75
+    },
+    {
+      title: 'Normal',
+      speed: 1
+    },
+    {
+      title: '1.25x',
+      speed: 1.25
+    },
+    {
+      title: '1.5x',
+      speed: 1.5
+    },
+    {
+      title: '2x',
+      speed: 2
+    }
+  ];
+});
+
+const validationRule = [
+  (value: string) => () : true | string => value.length > 0 || 'Required',
+  (value: string) => () : true | string => isNumber(value) || 'The input has to be a number',
+  (value: string) => () : true | string => Number.parseFloat(value) > 0 || 'The input has to be greater than zero.'
 ];
 </script>
``` 

I'd like to change '1x' to 'Normal', but I don't know how to add translation strings for all the languages.